plugins/python/regress/testdata/check_example_io_plugin_command_log.stored

```diff
@@ -0,0 +1,16 @@
+ -- Plugin STARTED --
+EXEC id --help
+EXEC info [
+    "command=/bin/id",
+    "runas_uid=0"
+]
+STD IN some standard input
+STD OUT some standard output
+STD ERR some standard error
+SUSPEND SIGTSTP (20)
```

Comment:
signal numbers are not portable so this test will fail on non-Linux systems

Reply:
fixed